core/commands/log.go

```diff
@@ -104,25 +103,39 @@ subsystems of a running daemon.
 	Type: stringList{},
 }
 
+const logLevelOption = "log-level"
+
 var logTailCmd = &cmds.Command{
 	Status: cmds.Experimental,
 	Helptext: cmds.HelpText{
 		Tagline: "Read the event log.",
 		ShortDescription: `
 Outputs event log messages (not other log messages) as they are generated.
```

Comment:
hm.. maybe clarify the limitation here:

```suggestion
Outputs event log messages (not other log messages) as they are generated.

NOTE: --log-level requires GOLOG_LOG_LEVEL to be lowered globally first.

Example:

  GOLOG_LOG_LEVEL="error,bitswap=debug" ipfs daemon
  ipfs log tail --log-level info

This will only return 'info' logs from bitswap and skip 'debug'.

```

Reply:
Done